redwood-layout-view/src/main/kotlin/app/cash/redwood/layout/view/ViewFlexContainer.kt

```diff
@@ -139,6 +141,11 @@ internal class ViewFlexContainer(
     }
   }
 
+  private fun newLinearLayout() = LinearLayout(context).apply {
+    orientation = if (direction.isHorizontal) LinearLayout.HORIZONTAL else LinearLayout.VERTICAL
```

Comment:
We only need orientation switching for scrolling. This could be a FrameLayout for the non-scrolling case as the HostView and FlexContainer take care of the orientation otherwise.

Reply:
I tried both replacing the `LinearLayout` with a `FrameLayout` and de-nesting completely to just return a `hostView`, but when doing that the views overlap each other.

Here's what it looks like when replacing [these lines](https://github.com/cashapp/redwood/blob/967b6f4ec2a531c5310b904e997cd8e12e1c9e84/samples/emoji-search/presenter/src/commonMain/kotlin/com/example/redwood/emojisearch/presenter/EmojiSearch.kt#L105-L116) with `repeat(100) { Text("Hello") }` using either a `FrameLayout` or de-nesting completely:

<img width="300" src="https://user-images.githubusercontent.com/6900601/214278300-ed2506cf-ed65-4b14-b957-c62c1bfdfd69.png"/>